app/resources/v1/location.js

```diff
@@ -0,0 +1,45 @@
+require('../../../lib/db.js')
+var Street = require('../../models/street.js')
```

Comment:
Let's use `const`s instead of `var`s where we can. The other modules haven't done it because we didn't get around to updating it.